src/main/java/com/cascadebot/cascadebot/ShutdownHandler.java

```diff
@@ -9,9 +9,13 @@
 import com.cascadebot.cascadebot.events.CommandListener;
 import com.cascadebot.cascadebot.tasks.Task;
 import com.cascadebot.shared.ExitCodes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class ShutdownHandler {
 
+    public static Logger logger = CascadeBot.logger;
```

Comment:
Don't assign a variable, just use it directly `CascadeBot.logger.info("")`

Reply:
of course